platform/broadcom/docker-syncd-brcm-rpc.mk

```diff
@@ -12,7 +12,7 @@ endif
 $(DOCKER_SYNCD_BRCM_RPC)_FILES += $(DSSERVE) $(BCMCMD) $(SUPERVISOR_PROC_EXIT_LISTENER_SCRIPT)
 $(DOCKER_SYNCD_BRCM_RPC)_LOAD_DOCKERS += $(DOCKER_SYNCD_BASE)
 SONIC_DOCKER_IMAGES += $(DOCKER_SYNCD_BRCM_RPC)
-SONIC_STRETCH_DOCKERS += $(DOCKER_SYNCD_BRCM_RPC)
+SONIC_BUSTER_DOCKERS += $(DOCKER_SYNCD_BRCM_RPC)
```

Comment:
As discussed in a previous comment, we do not reference a `SONIC_BUSTER_DOCKERS` variable. This line is unnecessary, correct, @lguohan?

Reply:
Done. Reverted the change.

Reply:
I think you should still delete the line, just not add a `SONIC_BUSTER_DOCKERS` line.